recipes/openfisca-core/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "OpenFisca-Core" %}
+{% set version = "35.7.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2e199a2a504625393f17dd26c363d3af1eda4f78a0470e40a87b2d9684bb0724
+
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - openfisca = openfisca_core.scripts.openfisca_command:main
+    - openfisca-run-test = openfisca_core.scripts.openfisca_command:main
+
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.6,<4.0
+    - dpath >=1.5.0,<3.0.0
+    - nptyping ==1.4.4
+    - numexpr >=2.7.0,<=3.0
+    - numpy >=1.11,<1.21
+    - psutil >=5.4.7,<6.0.0
+    - PyYAML >=3.10
+    - sortedcontainers ==2.2.2
+    - typing-extensions ==3.10.0.2
+
+
+
+test:
+  imports:
+    - openfisca_core
+    - openfisca_core.commons
+  requires:
+    - pytest
+    - pip
+  commands:
+    - openfisca --help
+    - openfisca-run-test --help
```

Comment:
did you try running `pip check` here? It would automatically check that your (strict) dependencies are compliant with the upstream `setup.py` definition.

Reply:
Oh yes, added it and add a new missing deps restriction :small_airplane: